.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+# Problem
+
+<!-- explain the problem this PR solves -->
+
+# Solution
+
+<!-- explain how this PR solves the problem -->
+
+### Checklist
+
+<!-- `x` completed boxes,  wrap tildes ~if unnecessary~ -->
+
+- [ ] Documentation updated
+- [ ] Tested code changes
+- [ ] Dependencies merged and published
```

Comment:
🎨 I tried to make this as concise as possible to avoid treating this list as white noise.